src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherAction.java

```diff
@@ -54,6 +55,14 @@ public ActionForward route(ActionMapping mapping, ActionForm form, HttpServletRe
         return forward;
     }
     
+    @Override
+    public ActionForward copy(ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response) throws Exception {
+    	RecurringDisbursementVoucherForm recurringForm = (RecurringDisbursementVoucherForm) form;
+    	recurringForm.getPdpStatuses().clear();
+    	ActionForward forward = super.copy(mapping, form, request, response);
```

Comment:
recurringForm.getPdpStatuses() calls getRecurringDisbursementVoucherDocumentService().findPdpStatuses((RecurringDisbursementVoucherDocument)getDocument()

And that looks at reucrring Details for DVs, and then finds the PDP statuses, so yes, we want to clear the DV document numbers from the recurring details